src/credentials/vue/root.vue

```diff
@@ -180,6 +191,30 @@ export default defineComponent({
         onIsAuthConnected(id: ServiceItemId, isConnected: boolean) {
             this.updateServiceLock(id, isConnected)
             this.renderItems()
+            // In some cases, during the connection process for one auth method,
+            // an already connected auth can be disconnected. This refreshes all
+            // auths to show the user the latest state of everything.
+            this.updateServiceConnections()
+        },
+        async updateServiceConnections() {
+            return Promise.all([
+                this.serviceItemsAuthStatus.RESOURCE_EXPLORER.isAuthConnected().then(isConnected => {
```

Comment:
Can we map `serviceItemsAuthStatus` instead of enumerating every key?

Reply:
Yup makes sense. I've added to my TODO list once I get the other services completed